sdk/identity/azure-identity/azure/identity/_credentials/workload_identity.py

```diff
@@ -57,6 +57,9 @@ class WorkloadIdentityCredential(ClientAssertionCredential, TokenFileMixin):
     :keyword str client_id: The client ID of a Microsoft Entra app registration.
     :keyword str token_file_path: The path to a file containing a Kubernetes service account token that authenticates
         the identity.
+    :keyword bool use_token_proxy: Whether or not to read token proxy configuration from environment variables and use
+        a token proxy to acquire tokens. If this value is True and proxy configuration isn't present or this value is
```

Comment:
Why we need both?

Can we take AZURE_KUBERNETES_TOKEN_PROXY is set as use_token_proxy is true?

Reply:
Discussed with Charles offline. Yes. Let's keep the argument but don't include it in the docstring now.